bgpd/bgp_attr.c

```diff
@@ -3610,6 +3868,34 @@ bgp_size_t bgp_packet_attribute(struct bgp *bgp, struct peer *peer,
 		}
 	}
 
+	/* SRv6 Service Information Attribute. */
+	if (afi == AFI_IP && safi == SAFI_MPLS_VPN) {
+		if (attr->srv6_l3vpn) {
```

Comment:
I understand there is no support for afi=AFI_IP6 and safi=SAFI_MPLS_VPN ?

Reply:
Yes. It desn't support AFI/SAFI=IPV6/VPNv6-unicast, at this time.
Supporting it is future work.